recipe/meta.yaml

```diff
@@ -1,70 +1,77 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "0.8.3.1" %}
+{% set sha256 = "4b0f5a5b4828915d973d3cbb86372aea9207e7b60fed912045d69242a74407f6" %}
+
 package:
-    name: ulmo
-    version: 0.8.2
+  name: ulmo
+  version: {{ 0.8.3.1 }}
```

Comment:
Sounds good. Just so I'm clear, conda-forge isn't actually running the test
suites right? just testing imports?

On Mon, Jul 18, 2016 at 4:58 PM Filipe <notifications@github.com> wrote:

> In recipe/meta.yaml
> <https://github.com/conda-forge/ulmo-feedstock/pull/2#discussion_r71238429>
> :
>
> >  package:
> > -    name: ulmo
> > -    version: 0.8.2
> > +  name: ulmo
> > +  version: {{ 0.8.3.1 }}
>
> Been there, done that.
>
> @dharhas <https://github.com/dharhas> it is fair game to self-merge
> updates like this when you are a maintainer. So once the CIs are green feel
> free to merge this. Thanks again for the new release and recipe update!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/conda-forge/ulmo-feedstock/pull/2/files/e0cd0250b2306392b1553ce5830c44d540f5502c#r71238429>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAiQlU3odQOId00vYwG-H3r5fI65lQqDks5qW_bpgaJpZM4JPKIl>
> .
>